docs/content/en/docs/installation/metrics.md

```diff
@@ -92,27 +105,40 @@ tetragon:
     metricsLabelFilter: "namespace,workload,binary" # "pod" label is disabled
 ```
 
-## Scrape metrics
+## Enable Prometheus ServiceMonitors
 
 Typically, metrics are scraped by Prometheus or another compatible agent (for example OpenTelemetry Collector), stored
 in Prometheus or another compatible database, then queried and visualized for example using Grafana.
 
-In Kubernetes, you can install Prometheus and Grafana using the `kube-prometheus-stack` Helm chart:
-
-```shell
-helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
-helm install kube-prometheus-stack prometheus-community/kube-prometheus-stack \
-  --namespace monitoring  --create-namespace \
-  --set prometheus.prometheusSpec.serviceMonitorSelectorNilUsesHelmValues=false
```

Comment:
hmm, I would leave this command here. It's helpful not only for users, but also for developers who want to check metrics in a local dev cluster.